float-features.lisp

```diff
@@ -218,12 +218,15 @@
                         when keyword collect `(si::trap-fpe ,keyword T))
                 NIL ,@body)
            (si::trap-fpe ,previous NIL))))
+    #+clasp
+     `(ext:with-float-traps-masked ,traps
```

Comment:
Does clasp really support all of the masks, including `:denormalized-operand`?

Reply:
no :denormalized-operand is not supported

Reply:
See comment below